### PR TITLE
Landing activity

### PR DESCRIPTION
**Pull request by sagarprasad63574**

### CODE REVIEW

**Comment by BorisMarin8004** [comment]
I think there are merge conflicts with main. In order to resolve it I think you need to checkout main, pull main, then checkout your branch, then `git rebase main`, then `git push -f`. Although I am not entirely sure if this will work, what do you think @larafonse?

**Comment by sagarprasad63574** [comment]
Oh I think I know why. Its because I created a java file with the same name as CreateActivity I can fix that

**Comment by sagarprasad63574** [comment]
Can you add this line to the build.gradle: implementation 'androidx.cardview:cardview:1.0.0'
and can you change android manifest from create activity to landing activity

**Comment by sagarprasad63574** [comment]
I added two new pull requests. Can you check it out. thanks

**Comment by BorisMarin8004** [comment]
You do not have to make new PRs you can just commit changes to this branch. This way they will be in the same PR.

**Comment by sagarprasad63574** [comment]
How? I can't commit changes from main in this PR. Wait, let me try again

**Comment by BorisMarin8004** [comment]
You do not need to do it to the main, you can just do it to this branch `Landing Activity`. However it seems that this branch is 3 commits ahead, 16 commits behind main. So I think you need to rebase it as I mentioned above

**Comment by sagarprasad63574** [comment]
Ok I will try

File: app/build.gradle [hunk 1]
```diff
@@ -14,15 +14,6 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    testOptions {
```
Comment (BorisMarin8004) [inline app/build.gradle]: @sagarprasad63574 Could you please run all tests, make sure they pass? I think we need these testOptions for some of them.

Comment (sagarprasad63574) [reply app/build.gradle]: ok

**Comment by sagarprasad63574** [comment]
ok the test passed @BorisMarin8004

**Review by BorisMarin8004 — APPROVED** [review]

**Review by Niko-113 — APPROVED** [review]
